closed/autoconf/custom-hook.m4

```diff
@@ -57,21 +57,26 @@ AC_DEFUN([OPENJ9_CONFIGURE_CMAKE],
       if test "x$with_cmake" == xyes -o "x$with_cmake" == x ; then
         with_cmake=cmake
       fi
-      if test "x$with_cmake" != xno ; then
-        if AS_EXECUTABLE_P(["$with_cmake"]) ; then
-          CMAKE="$with_cmake"
-        else
-          BASIC_REQUIRE_PROGS([CMAKE], [$with_cmake])
-        fi
-        with_cmake=yes
-      fi
     ],
-    [with_cmake=no])
-  if test "$with_cmake" == yes ; then
-    OPENJ9_ENABLE_CMAKE=true
-  else
-    OPENJ9_ENABLE_CMAKE=false
-  fi
+    [
+      if test "x$OPENJDK_BUILD_OS" = xmacosx ; then
+        with_cmake=cmake
+      else
+        with_cmake=no
+      fi
+    ])
+    # at this wpoint with_cmake should either be no, or the name of the cmake cmd
+    if test "$with_cmake" != no ; then
```

Comment:
typo: wpoint -> point
I suggest this `if` statement would be easier to follow if inverted (so `if` and `else` are closer to each other).